pom.xml

```diff
@@ -185,7 +185,7 @@
                         <configuration>
                             <referenceVersion>2.23ea0</referenceVersion>
                             <artifactsURI>https://teamcity.chronicle.software/repository/download</artifactsURI>
-                            <binaryCompatibilityPercentageRequired>99.9</binaryCompatibilityPercentageRequired>
+                            <binaryCompatibilityPercentageRequired>99.8</binaryCompatibilityPercentageRequired>
```

Comment:
Good to list the API changes in a comment here every time the % is reduced

Reply:
I'll do that in a commit message.